actions/lib/compress_archive_package.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+# Simple script that should be executed on a biotank when we want to compress a
+# runfolder to be archived to PDC via TSM. The runfolder to archive is expected
+# to be a folder full of symlinks pointing to real source folder. 
+
+set -e
+set -o pipefail 
+
+# The full path to the runfolder that we want to archive.
```

Comment:
Is there somekind of checking we could do here to make sure that the input is sane?